dbms/src/Server/PerformanceTest.cpp

```diff
@@ -711,11 +696,15 @@ class PerformanceTest
                 {
                     Connection::Packet packet = connection.receivePacket();
 
-                    if (packet.type == Protocol::Server::Data) {
+                    if (packet.type == Protocol::Server::Data)
+                    {
                         for (const ColumnWithTypeAndName & column : packet.block.getColumns())
                         {
-                            if (column.name == "result" && column.column->getDataAt(0).data != nullptr) {
-                                exist = column.column->get64(0);
+                            if (column.name == "result" && column.column->size() > 0)
+                            {
+                                Field field;
+                                column.column->get(0, field);
```

Comment:
it was fixed here - https://github.com/yandex/ClickHouse/commit/8c9aa37318c87cb0d226721e74fc450d50b03fb3